src/common/utils.tsx

```diff
@@ -369,3 +370,20 @@ export const getContractIdFromTx = (tx?: Transaction | MempoolTransaction) => {
   if (tx.tx_type === 'smart_contract') contractId = tx.smart_contract.contract_id;
   return contractId;
 };
+
+export function initBigNumber(num: string | number | BigNumber) {
+  return BigNumber.isBigNumber(num) ? num : new BigNumber(num);
+}
+
+export const ftDecimals = (value: number | string | BigNumber, decimals: number = STX_DECIMALS) => {
```

Comment:
I don't reckon this function should have the default parameter.

Reply:
Yeah, good point, prob not the best way to do it.